tools/eslint-plugin/dist/bin/no-internal-report.js

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env node
+
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+"use strict"
+const path = require('path');
+const fs = require('fs');
+
+function getNodeModulesDir(startPath) {
+  let currPath = startPath;
+  let baseName = path.basename(currPath);
+  while (baseName !== "node_modules") {
+    currPath = path.resolve(currPath, "..")
+    if (currPath === "/")
+      return undefined;
+    baseName = path.basename(currPath);
+  }
+  return currPath;
+}
+
+const nodeModules = getNodeModulesDir(path.dirname(process.argv[1]));
+if (!nodeModules)
+  throw ("Could not find node_modules directory");
+
+const distDir = path.join(nodeModules, "@bentley/eslint-plugin/dist")
+if (!fs.existsSync(distDir))
+  throw ("Could not find required dir: " + distDir);
+
+// Run eslint with the appropriate configuration and formatter to get a report of the no-internal rule
+let args = [
+  "-c", path.join(distDir, "configs/no-internal.js"),
```

Comment:
you might want to add --no-eslintrc here. By default, `-c` will merge the config with whatever is in the user's `package.json` or `eslintrc` file. So while the formatter won't output anything from other rules, this makes it slower than necessary.

Reply:
I've removed this file entirely and have added --no-eslintrc to the command line.